### ISSUE TITLE
py3status: error: too few arguments

### ISSUE THREAD

**Issue by lasers** [ultrabug/py3status#1692]
Running `py3status` alone or with arguments breaks on Python2 with `error: too few arguments`. 
Running `py3status` alone or with arguments works fine on Python3.

dbc3aea5579cae363c25af4cc8378a4e54a65d21 is the first bad commit
```
dbc3aea5579cae363c25af4cc8378a4e54a65d21 is the first bad commit
commit dbc3aea5579cae363c25af4cc8378a4e54a65d21
Author: lasers <lasers@users.noreply.github.com>
Date:   Sun Feb 3 14:29:00 2019 -0600

    py3status: transform modules and docstring commands (#1677)
```

Arguments in Python2.7 does not work because of recently implemented subparsers (i.e `list` and hidden `docstring`.). It looks like Python2.7 does not allow optional subparsers back in the days. See https://bugs.python.org/issue9253.

Solution.
# Python 3.7, when?

Alternative. Move all this to `py3-cmd`. I haz a coincidence branch too... [py3-cmd-list](https://github.com/ultrabug/py3status/tree/py3-cmd-list).

**Comment by lasers** [comment]
@ultrabug We should fix this for `Python2`. Critical bug. 

Our choices:
1) Enforce Python3, but not Python 3.6 or Python 3.7 yet.
2) Merge py3-cmd-list branch. It moves from `py3status` to `py3-cmd`.

**Comment by Cereal2nd** [comment]
can we please get this fixed? or merge in the py3 PR?

**Comment by lasers** [comment]
Closed via #1713.
